swim/scripts/init.js

```diff
@@ -154,6 +155,27 @@ Hooks.on(`ready`, () => {
     warpgate.event.watch("SWIM.updateCombat-previousTurn", gm_relay.combat_previousTurn, swim.is_first_gm)
     warpgate.event.watch("SWIM.updateCombat-nextTurn", gm_relay.combat_nextTurn, swim.is_first_gm)
     warpgate.event.watch("SWIM.updateCombat-currentTurn", gm_relay.combat_currentTurn, swim.is_first_gm)
+
+    //SWIM per-actor/item config header button
+    if (game.user.isGM) {
```

Comment:
```js
if (game.user.isGM || game.settings.get('swim', 'allowUserConfig') {
```
I will add the setting for that so GMs can decide whether or not they want to allow non-GMs to edit the config.

Reply:
You want me to do it or you do it in this branch?